corehq/apps/app_manager/views/modules.py

```diff
@@ -1037,6 +1052,85 @@ def _init_module_case_type(module):
         module.case_type = 'case'
 
 
+def _init_biometrics_enroll_module(app, lang):
```

Comment:
I wonder if there is a better place to put these 2 functions, as these don't seem like `view` utils or directly connected to the view in any way. Maybe a `biometrics` or `simprints` file as part of app manager?

Reply:
I'm going by Simprint's documentation: https://sites.google.com/simprints.com/simprints-for-developers/commcare/enrolment

I was also confused by the lack of the second L.